test/e2e_node/eviction_test.go

```diff
@@ -513,16 +512,13 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering", framework.WithSlow(), framewo
 		runEvictionTest(f, pressureTimeout, expectedNodeCondition, expectedStarvedResource, logPidMetrics, specs)
```

Comment:
It could, but the issue with the pre-existing test is that it is flaky, whereas the simplified test passes consistently,  see here: https://testgrid.k8s.io/sig-node-containerd#node-kubelet-containerd-eviction. It fails for reasons not related to DisruptionTarget condtion. There is an attempt to make it stable https://github.com/kubernetes/kubernetes/pull/123369, but it seems stale for now.